### ISSUE TITLE
GraphQL does not work with flowtype

### ISSUE THREAD

**Issue by Pfeifenjoy** [graphql/graphql-js#2320]
# Reporting issues with GraphQL.js

I am trying to use graphql (v14.4.8) together with flowtype (flow-bin v0.114.0).
However even without writing any actual code I always get the following error:

```
Cannot determine a type for object literal [1]. ObjMap [2] cannot be spread because the indexer
string [3] may overwrite properties with explicit keys in a way that Flow cannot track. Can you
spread ObjMap [2] first or remove the indexer?

     node_modules/graphql/utilities/extendSchema.js.flow
 [1] 283│       fields: () => ({
     284│         ...mapValue(config.fields, field => ({
     285│           ...field,
     286│           type: replaceType(field.type),
     287│         })),
     288│         ...keyValMap(
     289│           fieldNodes,
     290│           field => field.name.value,
     291│           field => astBuilder.buildInputField(field),
     292│         ),
     293│       }),
     294│       extensionASTNodes: config.extensionASTNodes.concat(extensions),
     295│     });

     node_modules/graphql/jsutils/ObjMap.js.flow
 [3]   3│ export type ObjMap<T> = { [key: string]: T, __proto__: null, ... };

     node_modules/graphql/jsutils/keyValMap.js.flow
 [2]  26│ ): ObjMap<V> {
```

This seems to be a type error of graphql.

**Comment by Pfeifenjoy** [comment]
Just noticed that this only occures if I also install `express-graphql`.

**Comment by Pfeifenjoy** [comment]
This seems to be fixed in the 15.0.0-alpha.2 version.
